src/app/core/models/common/export-settings.model.ts

```
@@ -87,6 +88,79 @@ export class ExportSettingModel {
+      ];
+    }
+
+    static dateGrouping(exportType: string, expenseGrouping: string, showApprovedDate: boolean, showVerificationDate: boolean): SelectFormOption[] {
+      // Determine the excluded date based on expenseGrouping
+      const excludedDate = expenseGrouping === ExpenseGroupingFieldOption.EXPENSE_ID
+        ? ExportDateType.LAST_SPENT_AT
+        : ExportDateType.SPENT_AT;
+
+      // Handle CCC export type
+      if (exportType === 'CCC') {
+        return this.getCreditCardExpenseGroupingDateOptions().filter(option => option.value !== excludedDate);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)